docs/source/content/overview/linear_SVM.rst

```diff
@@ -2,67 +2,157 @@
+You might be wondering that there could be multiple lines that split the data well. In fact there is a infinite
+amount of lines that can divide two classes.  As you can see in the graph below every line splits the squares and
+the circles, so which one do we choose?
 
-.. figure:: https://github.com/machinelearningmindset/machine-learning-for-everybody/blob/master/docs/source/content/overview/_img/Possible_hyperplane.PNG
-   :scale: 50 %
+.. figure:: _img/Possible_hyperplane.png
+   :scale: 50%
    :alt: Possible_Hyperplane
 
    Ref: https://towardsdatascience.com/support-vector-machine-introduction-to-machine-learning-algorithms-934a444fca47 
 
-We want to choose the line/plane with the **maximum margin**. Maximizing the margin will give us the optimal line to classify the data. This is shown in the figure below.  
+So how does SVM find the ideal line to separate the two classes. It doesn't just pick a random one. The algorithm 
+the line/hyperplane with the **maximum margin**. Maximizing the margin will give us the optimal line to classify the data. 
+This is shown in the figure below.  
 
-.. figure:: https://github.com/machinelearningmindset/machine-learning-for-everybody/blob/master/docs/source/content/overview/_img/Optimal_hyperplane.PNG
-   :scale: 50 %
+.. figure:: _img/Optimal_hyperplane.PNG
+   :scale: 50%
    :alt: Optimal_Hyperplane
 
    Ref: https://towardsdatascience.com/support-vector-machine-introduction-to-machine-learning-algorithms-934a444fca47 
 
 How to maximize the margin?
 ---------------------------
 
-The data that is closest to the line are what determmine the optimal line. These data points are called **support vectors**. They are shown as the filled in squares and circles above. There is a lot of complex math that goes into finding the support vectors and maximizing the margin. We won't go into that, we just want to get the basic idea behind SVMs. 
+The data that is closest to the line are what determmine the optimal line. These data points are called 
+**support vectors**. They are shown as the filled in squares and circles above. The distance from these vectors to the
+hyperplane is called the **margin**. In general the further those points are from the hyperplane, the greater the 
+probability of correctly classifying the data. There is a lot of complex math that goes into finding the support vectors
+and maximizing the margin. We won't go into that, we just want to get the basic idea behind SVMs. 
 
 Ignore Outliers
 ---------------
 
-Sometimes data classes will have **outliers**, (data that is separated from it's class). Support Vector Machines will ignore these outliers. This is shown in the figure below. 
+Sometimes data classes will have **outliers**. This is data points that are clearly separated from the rest of it's class.
```

Comment:
`These are` data points that are clearly separated from the rest of `their` class